%YAML:1.0
CameraInfo:
  -
    Name: usbcam2f1_l
    DevID: 5
    Width: 640
    Height: 480
    PixelFormat: MJPG
    NRotate90: 0
  -
    Name: usbcam2f1_r
    DevID: 6
    Width: 640
    Height: 480
    PixelFormat: MJPG
    NRotate90: 0
